runs/run_taylor_fwd_inv.py

```diff
@@ -0,0 +1,99 @@
+# For fenics_ice copyright information see ACKNOWLEDGEMENTS in the fenics_ice
+# root directory
+
+# This file is part of fenics_ice.
+#
+# fenics_ice is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# fenics_ice is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with tlm_adjoint.  If not, see <https://www.gnu.org/licenses/>.
+
+# This script does not output anything. It is only use for
+# running taylor test verification on the forward runs outside
+# of the pytest environment
+
+import os
+os.environ["OMP_NUM_THREADS"] = "1"
+os.environ["OPENBLAS_NUM_THREADS"] = "1"
+
+import sys
+
+from runs import run_forward
+
+from fenics_ice.backend import clear_caches, compute_gradient, \
+    reset_manager, stop_manager, taylor_test
+
+def EQReset():
+    """Take care of tlm_adjoint EquationManager"""
+    # This prevents checkpointing errors when these run phases
+    # are tested after the stuff in test_model.py
+    reset_manager("memory")
+    clear_caches()
+    stop_manager()
+
+def test_tv_run_forward(config_file):
+
+    EQReset()
+
+    mdl_out = run_forward.run_forward(config_file)
+
+    slvr = mdl_out.solvers[0]
+
+    qoi_func = slvr.get_qoi_func()
+    cntrl = slvr.get_control()
+
+    slvr.reset_ts_zero()
+    J = slvr.timestep(adjoint_flag=1, qoi_func=qoi_func)[0]
+    dJ = compute_gradient(J, cntrl)
+
+    def forward_ts(cntrl, cntrl_init, name):
+        slvr.reset_ts_zero()
+        if (name == 'alpha'):
+            slvr._alpha = cntrl
+        elif (name == 'beta'):
+            slvr._beta = cntrl
+        else:
+            raise ValueError(f"Unrecognised cntrl name: {name}")
+
+        result = slvr.timestep(adjoint_flag=1, qoi_func=slvr.get_qoi_func())[0]

```

Comment:
This tests the first functional, which may be the initial functional, and I don't think this depends on `alpha`

Reply:
changing things to [1]  ``result = slvr.timestep(adjoint_flag=1, qoi_func=slvr.get_qoi_func())[1]``

gave me this: 
```
Time for solve: 1.4720582962036133
Error norms, no adjoint   = [13010.7027514   6453.38067961  3213.66630173  1603.57322049
   800.97114038   400.28164792]
INFO:tlm_adjoint.verification:Error norms, no adjoint   = [13010.7027514   6453.38067961  3213.66630173  1603.57322049
   800.97114038   400.28164792]
Orders,      no adjoint   = [1.01157185 1.005835   1.0029299  1.00146806 1.00073479]
INFO:tlm_adjoint.verification:Orders,      no adjoint   = [1.01157185 1.005835   1.0029299  1.00146806 1.00073479]
Error norms, with adjoint = [2.08217819e+02 5.21382135e+01 1.30450687e+01 3.26260397e+00
 8.15832123e-01 2.03993790e-01]
INFO:tlm_adjoint.verification:Error norms, with adjoint = [2.08217819e+02 5.21382135e+01 1.30450687e+01 3.26260397e+00
 8.15832123e-01 2.03993790e-01]
Orders,      with adjoint = [1.99768048 1.99883661 1.99940876 1.99967966 1.99974708]
INFO:tlm_adjoint.verification:Orders,      with adjoint = [1.99768048 1.99883661 1.99940876 1.99967966 1.99974708]
Forward simulation cntrl: alpha min_order: 1.9976804835434803
1.9976804835434803
```